src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -781,9 +781,30 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
 
             PackagePathsViewModel.RootLocations.CollectionChanged += PackagePathsViewModel_RootLocations_CollectionChanged;
 
+            WorkspaceEvents.WorkspaceSettingsChanged += PreferencesViewModel_WorkspaceSettingsChanged;
```

Comment:
if we do end up fixing it this way we should clean this up on shutdown, I think @sm6srw just added a spot where the other eventHandlers here are unsubscribed.

Reply:
Agreed

Reply:
Yes, PreferencesViewModel:UnsubscribeAllEvents() can be used for that purpose.

Reply:
@QilongTang @mjkkirschner 
unsubscribing the event in the next commit: https://github.com/DynamoDS/Dynamo/pull/11845/commits/e4e1a7e52bfd42bf592c6be18222c1ae41caad8c